.github/workflows/ghpages.yml

```diff
@@ -0,0 +1,41 @@
+name: GH Pages
+
+on: push
```

Comment:
Please add a filter to only run on `master` and add the regular `pull_request` trigger, otherwise it won't run if the PR is made from a fork.

Reply:
Done